RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -899,7 +899,7 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
     <value>Versionsverwaltung</value>
   </data>
   <data name="SourceControl_AttributesFile" xml:space="preserve">
-    <value>Attributes File</value>
+    <value>'.gitattributes' Datei</value>
```

Comment:
This one needs "domain knowledge" over the workings of git. Does a Rubberduck Source Control user need that @ckuhn203 ?